Task08/SomeSubject.java

```diff
@@ -0,0 +1,61 @@
+package test.HW8;
+
+/**
+ * Class containing fields of products
+ */
+public class SomeSubject {
+    private static int countOfAllSubjects = 0;
+    private static double priceOfAllSubjects = 0;
+    private static int countOfTypes = 0;
+    private String typeOfEntity;
+    private String nameOfSubject;
+    private int countOfSubject;
+    private double priceOfSubject;
+
+    /**
+     * Constructor for TYPE of product.
+     * Summarizes the number of calls.
+     *
+     * @param name Name of type
+     */
+    SomeSubject(String name) {
+        typeOfEntity = name;
+        countOfTypes++;
+    }
+
+    /**
+     * Constructor for product.
+     * Summarizes the number of calls and price all products.
+     *
+     * @param name  Name of product
+     * @param count Count of product
+     * @param price Price of product
+     */
+    SomeSubject(String name, int count, int price) {
+        nameOfSubject = name;
+        countOfSubject = count;
+        priceOfSubject = price;
+        countOfAllSubjects++;
```

Comment:
It won't work. Let think about it and try to understand how it should be re-written